scala/lambdas/ingest-mapper/src/test/scala/uk/gov/nationalarchives/ingestmapper/MetadataServiceTest.scala

```diff
@@ -92,22 +92,33 @@ class MetadataServiceTest extends AnyFlatSpec with MockitoSugar with TableDriven
             }
 
           val batchId = "batchId"
-          val folderId = UUID.randomUUID()
-          val assetId = UUID.randomUUID()
-          val fileIdOne = UUID.randomUUID()
-          val fileIdTwo = UUID.randomUUID()
+          val folderIdOne = UUID.randomUUID()
+          val assetIdOne = UUID.randomUUID()
+          val fileIdOneOne = UUID.randomUUID()
+          val fileIdOneTwo = UUID.randomUUID()
+          val folderIdTwo = UUID.randomUUID()
+          val assetIdTwo = UUID.randomUUID()
+          val fileIdTwoOne = UUID.randomUUID()
+          val fileIdTwoTwo = UUID.randomUUID()
           val departmentTableItem = tableItem(departmentId, "department", "")
           val seriesTableItem = seriesIdOpt.map(id => tableItem(id, "series", departmentId.toString))
           val departmentAndSeries = DepartmentAndSeriesTableItems(departmentTableItem, seriesTableItem)
 
           val expectedTimeInSecs = 1712707200
           val originalFileId = UUID.randomUUID()
-          val originalMetadataFileId = UUID.randomUUID()
+          val originalMetadataFileIdOne = UUID.randomUUID()
+          val originalMetadataFileIdTwo = UUID.randomUUID()
           val metadata =
-            s"""[{"id":"$folderId","parentId":null,"title":"TestTitle","type":"ArchiveFolder","name":"TestName","fileSize":null, "series": null},
-           |{"id":"$assetId","parentId":"$folderId","title":"TestAssetTitle","type":"Asset","name":"TestAssetName","fileSize":null, "originalFiles" : ["$originalFileId"], "originalMetadataFiles": ["$originalMetadataFileId"], "customMetadataAttribute1": "customMetadataAttributeValue"},
-           |{"id":"$fileIdOne","parentId":"$assetId","title":"Test","type":"File","name":"$name","fileSize":1, "checksumSha256": "$name-checksum"},
-           |{"id":"$fileIdTwo","parentId":"$assetId","title":"","type":"File","name":"TEST-metadata.json","fileSize":2, "checksumSha256": "metadata-checksum"}]
+            s"""[
+           |{"id":"$folderIdTwo","parentId":null,"title":"TestTitle2","type":"ArchiveFolder","name":"TestName2","fileSize":null, "series": null},
+           |{"id":"$assetIdTwo","parentId":"$folderIdTwo","title":"TestAssetTitle2","type":"Asset","name":"TestAssetName2","fileSize":null, "originalFiles" : ["$originalFileId"], "originalMetadataFiles": ["$originalMetadataFileIdTwo"], "customMetadataAttribute1": "customMetadataAttributeValue"},
+           |{"id":"$fileIdTwoOne","parentId":"$assetIdOne","title":"Test2","type":"File","name":"$name","fileSize":1, "checksumSha256": "$name-checksum"},
```

Comment:
Isn't the parentId supposed to be `assetIdTwo`?

Reply:
Good spot, thanks.